docs/node-operators/how-to/maintain-multisig-contract.md

```diff
@@ -0,0 +1,71 @@
+---
+sidebar_position: 10
+title: Maintaining the Multisig Contract
+sidebar_label: Maintaining Multisig Contract
+hide_title: false
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+The [Multisig Control](../../api/bridge/interfaces/IMultisigControl.md) contract contains a set of signers whose signatures are required to complete actions on the Ethereum Bridge. The set of signers on the contract should match the set of consensus validators in the network. As validators join or leave the network and the set of consensus validators on the Vega network changes, the signer set on the Multisig Control contract must be updated to alongside those changes to maintain the security of the bridge.
+
+## Identifying when to update the contract
+
+The consensus set of validators on the Vega network can change because of the following:
+* The amounts staked to validators change such that a consensus validator is demoted and replaced by a standby validator with more stake
+* A consensus validator is showing poor performance and they are demoted and replaced by a more performant standby validator
+* The value of the network parameter `network.validators.tendermint.number` changes causing the size of the consensus set to increase or decrease
```

Comment:
```suggestion
* The value of the network parameter <NetworkParameter frontMatter={frontMatter} param="network.validators.tendermint.number" /> changes causing the size of the consensus set to increase or decrease
```

This will render like the first example on [this page](https://docs.vega.xyz/testnet/concepts/vega-protocol#example), with the network param name and value. Using the frontmatter property for the network, we make sure it's kept reasonably up to date - and it links through to the explorer.

Reply:
@edd I think for this one it would be good to hide the value, what do you think?

Reply:
@candida-d I have updated the suggestion to hide the value